lib/src/components/HalTable.jsx

```diff
@@ -109,24 +106,26 @@ const HalTable = ({ collectionUrl, asyncHeadersHandler, headers, columns, itemsP
     <DxcHALTableContainer>
       <DxcTable>
         <HeaderRow>
```

Comment:
Why this `HeaderRow` has a height of `60px`? Is it in the desgin?

Reply:
I think it uses the same design as the [Resultset Table](https://developer.dxc.com/halstack/next/components/resultset-table/specifications/).

Reply:
So, in that case, there are some values that are not matching the design. 
- Height of the cells: as it is specified, it should be 60px but in the component is less.

Component:
![image](https://user-images.githubusercontent.com/34685461/196197709-8e8be229-90f9-4fbe-8b22-26d6df2d003b.png)

Design:
![image](https://user-images.githubusercontent.com/34685461/196197879-b045d718-0af1-45bb-a2a0-b8ff014ec8bc.png)

- Paddings in the cells: the last cell of the row should have a `padding-right: 40px`.

Component:
![image](https://user-images.githubusercontent.com/34685461/196199681-9025255c-b906-4099-977e-db7afc79b9d9.png)

Design:
![image](https://user-images.githubusercontent.com/34685461/196198258-5b62974b-b3f8-4794-bbca-2e7e712c365b.png)

- I would put tab index for the column name when it has sorting. Also some space between label and icon.

Component:
![image](https://user-images.githubusercontent.com/34685461/196200827-14a246b0-e8b3-4a7b-bb6b-d4ce5b93afd9.png)

Design:
![image](https://user-images.githubusercontent.com/34685461/196200368-ea57217f-03e4-4430-8941-46d84a9df38e.png)